nifi-docker/dockermaven/sh/start.sh

```diff
@@ -33,15 +33,29 @@ if [ ! -z "${NIFI_JVM_DEBUGGER}" ]; then
 fi
 
 # Establish baseline properties
-prop_replace 'nifi.web.http.port'               "${NIFI_WEB_HTTP_PORT:-8080}"
-prop_replace 'nifi.web.http.host'               "${NIFI_WEB_HTTP_HOST:-$HOSTNAME}"
+prop_replace 'nifi.web.https.port'              "${NIFI_WEB_HTTPS_PORT:-8443}"
+prop_replace 'nifi.web.https.host'              "${NIFI_WEB_HTTPS_HOST:-$HOSTNAME}"
 prop_replace 'nifi.remote.input.host'           "${NIFI_REMOTE_INPUT_HOST:-$HOSTNAME}"
 prop_replace 'nifi.remote.input.socket.port'    "${NIFI_REMOTE_INPUT_SOCKET_PORT:-10000}"
-prop_replace 'nifi.remote.input.secure'         'false'
+prop_replace 'nifi.remote.input.secure'         'true'
 
 # Set nifi-toolkit properties files and baseUrl
 "${scripts_dir}/toolkit.sh"
-prop_replace 'baseUrl' "http://${NIFI_WEB_HTTP_HOST:-$HOSTNAME}:${NIFI_WEB_HTTP_PORT:-8080}" ${nifi_toolkit_props_file}
+prop_replace 'baseUrl' "https://${NIFI_WEB_HTTPS_HOST:-$HOSTNAME}:${NIFI_WEB_HTTPS_PORT:-8443}" ${nifi_toolkit_props_file}
+
+if [ ! -z "${NIFI_WEB_HTTP_PORT}" ]; then
+    prop_replace 'nifi.web.https.port'                        ''
+    prop_replace 'nifi.web.https.host'                        ''
+    prop_replace 'nifi.web.http.port'                         "${NIFI_WEB_HTTP_PORT}"
+    prop_replace 'nifi.web.http.host'                         "${NIFI_WEB_HTTP_HOST:-$HOSTNAME}"
+    prop_replace 'nifi.remote.input.secure'                   'false'
```

Comment:
Should also set `nifi.cluster.protocol.is.secure` (I think this is a historical omission but makes sense to correct)

These comments apply to both sets of the docker start scripts

Reply:
Yes, this is a good opportunity to make the adjustment.

Reply:
Changes look good (I can't resolve this as a thread for some reason)

Reply:
Thanks again @ChrisSamo632, I appreciate the thorough review!